sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -1611,7 +1611,9 @@ def wait_until_finish(self, duration=None):
     if not self.is_in_terminal_state():
       if not self.has_job:
         raise IOError('Failed to get the Dataflow job id.')
-
+      consoleUrl = """Console URL: 'https://console.cloud.google.com/
+          dataflow/jobs/<RegionId>/{}?project=
```

Comment:
These empty spaces will be concatenated into triple-quoted string. Use + or ('literal 1 with space at the end ' 'literal 2 ' ...)